mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/EventBusIdTest.java

```diff
@@ -41,6 +43,26 @@ void ofShouldDeserializeUUIDs() {
         assertThat(EventBusId.of(UUID_1.toString()))
             .isEqualTo(EventBusId.of(UUID_1));
     }
+    @Test
+    void oo() {
```

Comment:
`oo` ?

Reply:
Debugging statement that got lost. Sorry.